script/install.sh

```diff
@@ -44,8 +44,24 @@ if [ -z "$1" ] || [ "$1" == "sgx" ]; then
 fi
 # RISC0
 if [ -z "$1" ] || [ "$1" == "risc0" ]; then
-	cargo install cargo-risczero
-	cargo risczero install --version v2024-02-08.1
+    echo "Current TERM: $TERM"
+    if [ -z "$TERM" ] || [ "$TERM" = "dumb" ]; then
+        # Set TERM to xterm-color256
+        echo "Setting TERM to xterm"
+        export TERM=xterm
+    fi
```

Comment:
more fancy output?

Reply:
No, that's because the rzup raises a error if no TERM set